### ISSUE TITLE
outputs in project.json missing

### ISSUE THREAD

**Issue by nelsond6** [khalilou88/jnxplus#30]
First, awesome project by the way.  Thanks for working on this project.  
When following your maven [tutorial](https://dev.to/gridou/announcing-jnxplus-nx-boot-maven-4g28), I noticed when doing `nx build my-lib` and if the build was pulled from nx cache, the actual target folder would not be re-created. After looking at the nx [documentation](https://nx.dev/configuration/projectjson), it looks like you can add an outputs property to the project.json build target.  So in my case, I added the following to the my-lib project.json file:
```
"build": {
      "executor": "@jnxplus/nx-boot-maven:build",
      "outputs": ["libs/my-lib/target"]
    }
```
Now after running `nx build my-lib`, and if nx decides to pull from cache, the my-lib target folder gets pulled from cache.  Do you see any issue in introducing this outputs property?

**Comment by khalilou88** [comment]
Hi @nelsond6 
Thanks for pointing this. I was not aware.
Since you know the solution, could you please create a PR?

**Comment by nelsond6** [comment]
@khalilou88 I am curious on how you test your plugin against an actual NX workspace while you develop.  Let me know what you have been doing as I don't have any experience with developing NX plugins.

**Comment by khalilou88** [comment]
> @khalilou88 Sure thing. I just pushed up the changes for maven. Your Gradle plugin support probably needs the same thing. I will try and do the same for Gradle.

The gradle work without this change and I don't know why

> @khalilou88 I am curious on how you test your plugin against an actual NX workspace while you develop. Let me know what you have been doing as I don't have any experience with developing NX plugins.

- I try to write some e2e tests to reproduce or tell what I want to do, tests should fails
- Write the code to fix or add the feature and execute build nx-boot-maven 
- Rerun the e2e tests and they should work

I use the tmp/nx-e2e/proj to see what happening and to run executors and generators also..

You can watch this video for more information:https://www.youtube.com/watch?v=fC1-4fAZDP4&t=184s

**Comment by khalilou88** [comment]
@nelsond6 Thank you for your work.
Your change is published within the version 1.6.0.
